components/net/http_loader.rs

```diff
@@ -422,7 +422,7 @@ pub struct StreamedResponse<R: HttpResponse> {
 impl<R: HttpResponse> Read for StreamedResponse<R> {
     #[inline]
     fn read(&mut self, buf: &mut [u8]) -> io::Result<usize> {
-        match self.decoder {
+       match self.decoder {
```

Comment:
yep, whoops, I caught and fixed that before, but then I must've accidentally undone it when I was trying to fix a different mistake I made. for some reason test-tidy doesn't catch it (assuming it's supposed to be unhappy about spaces not divisible by 4)